common/persistence/src/main/java/app/coronawarn/server/common/persistence/repository/DiagnosisKeyRepository.java

```diff
@@ -39,6 +39,27 @@
   @Query("SELECT COUNT(*) FROM diagnosis_key WHERE submission_timestamp<=:threshold")
   int countOlderThanOrEqual(@Param("threshold") long submissionTimestamp);
 
+  /**
+   * Counts all entries that have a submission timestamp less or equal than the specified one
+   *  and match the origin country_code.
```

Comment:
was the origin word  intended in the doc , or is the meaning more like "and match the given country_code"

Reply:
The meaning was supposed to be something like "match the given country_code", yes. I will rephrase this.